### DISCUSSION TITLE
Gitter error Something went wrong and Element was unable to load.

### DISCUSSION THREAD

**Discussion by debiandragon** [community/community#55315] [New to GitHub]
new to github.  visited a project gitter page yesterday to read discussions.  today gitter will not load.  platform error is : Something went wrong and Element was unable to load.

reloaded page to no success
similar issue experienced with firefox and chrome (recent versions)

any suggestions?

**Comment by queenofcorgis** [comment]
Hi there - are you still experiencing this? There may have been an outage that caused this.

**Comment by debiandragon** [comment]
can slow connection speed prevent gitter from loading/cause the error in original post ?  ip blocking ?  i was able to finally get the gitter page to load today, but from home computer it is still displaying error in original post = there may be more causes for the problem.  just want to narrow down cause so i can reach page on home computer as well.
